src/GitHub.Logging/Logging/LogManager.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.IO;
+using GitHub.Info;
+using Serilog;
+using Serilog.Core;
+
+namespace GitHub.Logging
+{
+    public static class LogManager
+    {
+        static Logger CreateLogger()
+        {
+            var logPath = Path.Combine(
+                Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
+                ApplicationInfo.ApplicationName,
+                "extension.log");
+
+            const string outputTemplate =
+                "{Timestamp:yyyy-MM-dd HH:mm:ss.fff}|{Level}|Thread:{ThreadId}|{SourceContext}|{Message}{NewLine}{Exception}";
+
+            return new LoggerConfiguration()
+                .Enrich.WithThreadId()
+                .MinimumLevel.Warning()
```

Comment:
When I change this to `.MinimumLevel.Informational()`, the `extension.log` gets created but remains blank. I'd expect at least something to show up?

For this PR, should this default to `.Informational()` for consistency our previous logging? We could change it in another PR if we decide we're logging too much.

Reply:
If I change that to `MinimumLevel.Informational()` and change TE context then I get entries logged, e.g.:

```
2017-09-12 19:16:38.329|Information|Thread:1|GitHub.VisualStudio.TeamExplorer.Connect.GitHubConnectSection|"IsBusy" title:"Local Git Repositories" busy:True
2017-09-12 19:16:38.344|Information|Thread:1|GitHub.VisualStudio.TeamExplorer.Connect.GitHubConnectSection|"IsBusy" title:"Local Git Repositories" busy:True
2017-09-12 19:16:38.474|Information|Thread:1|GitHub.VisualStudio.TeamExplorer.Connect.GitHubConnectSection|"IsBusy" title:"Local Git Repositories" busy:True
2017-09-12 19:16:38.474|Information|Thread:1|GitHub.VisualStudio.TeamExplorer.Connect.GitHubConnectSection|"IsBusy" title:"Local Git Repositories" busy:True
2017-09-12 19:16:38.485|Information|Thread:1|GitHub.VisualStudio.TeamExplorer.Connect.GitHubConnectSection|"Title" title:"Local Git Repositories (30)" busy:True
2017-09-12 19:16:38.485|Information|Thread:1|GitHub.VisualStudio.TeamExplorer.Connect.GitHubConnectSection|"Title" title:"Local Git Repositories (30)" busy:True
2017-09-12 19:16:38.486|Information|Thread:1|GitHub.VisualStudio.TeamExplorer.Connect.GitHubConnectSection|"IsBusy" title:"Local Git Repositories (30)" busy:False
2017-09-12 19:16:38.487|Information|Thread:1|GitHub.VisualStudio.TeamExplorer.Connect.GitHubConnectSection|"IsBusy" title:"Local Git Repositories (30)" busy:False
2017-09-12 19:16:38.654|Information|Thread:1|GitHub.VisualStudio.TeamExplorer.Connect.GitHubConnectSection|"IsBusy" title:"Local Git Repositories (30)" busy:False
2017-09-12 19:16:38.654|Information|Thread:1|GitHub.VisualStudio.TeamExplorer.Connect.GitHubConnectSection|"IsBusy" title:"Local Git Repositories (30)" busy:False
2017-09-12 19:16:41.807|Information|Thread:1|GitHub.VisualStudio.TeamExplorer.Connect.GitHubConnectSection|"IsBusy" title:"Local Git Repositories (30)" busy:True
2017-09-12 19:16:41.807|Information|Thread:1|GitHub.VisualStudio.TeamExplorer.Connect.GitHubConnectSection|"IsBusy" title:"Local Git Repositories (30)" busy:True
```

Because changing repositories causes [this debug logging statement](https://github.com/github/VisualStudio/blob/3b7535a3c4a2e5b43cd45954aac98600a9a862c9/src/GitHub.TeamFoundation.14/Connect/GitHubConnectSection.cs#L523) to be hit.

Are you sure you're hitting any `log.Information` statements?

It might be worth defaulting it to `.Informational()` I guess - what particular information do you want to be visible in user logs?

Reply:
Maybe I'm not actually hitting anything. I thought there was a fair bit of info logging, but maybe not as much as I expected.

As much as anything, I want to see what information we're writing to the log. If anything does go wrong, it might be interesting to see what else has been happening.